examples/step-13/step-13.cc

```diff
@@ -1239,50 +1203,50 @@ namespace Step13
-                       const Quadrature<dim>    &quadrature,
-                       const Function<dim>      &rhs_function,
-                       const Function<dim>      &boundary_values);
+      RefinementKelly(Triangulation<dim> &      coarse_grid,
+                      const FiniteElement<dim> &fe,
+                      const Quadrature<dim> &   quadrature,
+                      const Function<dim> &     rhs_function,
+                      const Function<dim> &     boundary_values);
 
-      virtual void refine_grid () override;
+      virtual void refine_grid() override;
     };
 
 
 
     template <int dim>
-    RefinementKelly<dim>::
-    RefinementKelly (Triangulation<dim>       &coarse_grid,
-                     const FiniteElement<dim> &fe,
-                     const Quadrature<dim>    &quadrature,
-                     const Function<dim>      &rhs_function,
-                     const Function<dim>      &boundary_values)
-      :
-      Base<dim> (coarse_grid),
-      PrimalSolver<dim> (coarse_grid, fe, quadrature,
-                         rhs_function, boundary_values)
+    RefinementKelly<dim>::RefinementKelly(
+      Triangulation<dim> &      coarse_grid,
+      const FiniteElement<dim> &fe,
+      const Quadrature<dim> &   quadrature,
+      const Function<dim> &     rhs_function,
+      const Function<dim> &     boundary_values) :
+      Base<dim>(coarse_grid),
+      PrimalSolver<dim>(coarse_grid,
+                        fe,
+                        quadrature,
+                        rhs_function,
+                        boundary_values)
     {}
 
 
 
     template <int dim>
-    void
-    RefinementKelly<dim>::refine_grid ()
+    void RefinementKelly<dim>::refine_grid()
     {
-      Vector<float> estimated_error_per_cell (this->triangulation->n_active_cells());
-      KellyErrorEstimator<dim>::estimate (this->dof_handler,
-                                          QGauss<dim-1>(3),
-                                          typename FunctionMap<dim>::type(),
-                                          this->solution,
-                                          estimated_error_per_cell);
-      GridRefinement::refine_and_coarsen_fixed_number (*this->triangulation,
-                                                       estimated_error_per_cell,
-                                                       0.3, 0.03);
-      this->triangulation->execute_coarsening_and_refinement ();
+      Vector<float> estimated_error_per_cell(
+        this->triangulation->n_active_cells());
+      KellyErrorEstimator<dim>::estimate(this->dof_handler,
+                                         QGauss<dim - 1>(3),
+                                         typename FunctionMap<dim>::type(),
+                                         this->solution,
+                                         estimated_error_per_cell);
+      GridRefinement::refine_and_coarsen_fixed_number(
+        *this->triangulation, estimated_error_per_cell, 0.3, 0.03);
```

Comment:
This is an example where a human clearly thought about the arrangement of arguments. It used to be
```
GridRefinement::refine_and_coarsen_fixed_number (*this->triangulation,
                                                       estimated_error_per_cell,
                                                       0.3, 0.03);￼
```
This clearly separates like arguments onto lines of their own. It makes it easier to read in my opinion -- and definitely easier to teach.

Reply:
By setting `PenaltyBreakBeforeFirstCallParameter: 90` we can discourage clang-format from doing that. There is no way of keeping one argument on the same line, though.

Actually I would really prefer increasing this penalty. Random example (before after):
```diff
-    VectorTools::interpolate_boundary_values(
-      dof_handler, 0, Functions::ZeroFunction<dim>(), boundary_values);
-    MatrixTools::apply_boundary_values(
-      boundary_values, system_matrix, newton_update, system_rhs);
+    VectorTools::interpolate_boundary_values(dof_handler,
+                                             0,
+                                             Functions::ZeroFunction<dim>(),
+                                             boundary_values);
+    MatrixTools::apply_boundary_values(boundary_values,
+                                       system_matrix,
+                                       newton_update,
+                                       system_rhs);

```